DOCUMENTATION.md

```diff
@@ -1,810 +1,807 @@
 
-|`error.code`|`error.message`|
-|:---|:---|
-|`429`|`[{`<br>`"error": "RateLimitError",`<br>`"message": "Exceeded rate limit for key type TEAM of 10 requests per 10 seconds"`<br>`}]`|
-|`429`|`[{`<br>`"error": "TooManyRequestsError",`<br>`"message": "Exceeded send limits (50) for today"`<br>`}]`|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient using a team-only API key"`<br>`]}`|
-|`400`|`[{`<br>`"error": "BadRequestError",`<br>`"message": "Can"t send to this recipient when service is in trial mode - see https://www.notifications.service.gov.uk/trial-mode"`<br>`}]`|
+```ruby
+personalisation: {
+  name: "John Smith",
+  ID: "300241",                      
+}
+```
 
-</details>
+You can leave out this argument if a template does not have any placeholder fields for personalised information.
 
-#### Arguments
+#### reference (optional)
 
-<details>
-<summary>
-Click here to expand for more information.
-</summary>
+A unique identifier. This reference identifies a single unique notification or a batch of notifications.
 
-##### `phone_number`
+```ruby
+reference: "your_reference_string",
+```
 
-The phone number of the recipient, only required for sms notifications.
+You can leave out this argument if you do not have a `reference`.
 
-##### `template_id`
+#### sms_sender_id (optional)
 
-Find by clicking **API info** for the template you want to send.
+A unique identifier of the sender of the text message notification. To find this information, go to the __Text Message sender__ settings screen:
 
-##### `reference`
+1. Sign in to your GOV.UK Notify account.
+1. Go to __Settings__.
+1. If you need to change to another service, select __Switch service__ in the top right corner of the screen and select the correct one.
+1. Go to the __Text Messages__ section and select __Manage__ on the __Text Message sender__ row.
 
-An optional identifier you generate. The `reference` can be used as a unique reference for the notification. Because Notify does not require this reference to be unique you could also use this reference to identify a batch or group of notifications.
+In this screen, you can then either:
 
-You can omit this argument if you do not require a reference for the notification.
+  - copy the sender ID that you want to use and paste it into the method
+  - select __Change__ to change the default sender that the service will use, and select __Save__
 
-##### `personalisation`
+```ruby
+sms_sender_id: "8e222534-7f05-4972-86e3-17c5d9f894e2",
+```
 
-If a template has placeholders, you need to provide their values, for example:
+You can leave out this argument if you do not have have an `sms_sender_id`.
 
-```python
-personalisation={
-    'first_name': 'Amala',
-    'reference_number': '300241',
-}
+### Response
+
+If the request to the client is successful, the client will return a `Notifications::Client:ResponseNotification`:
+
+```ruby
+ @content={'notification content'},
```

Comment:
With all of the responses it will be clearer to keep the format similar to what we had originally, so something like this- 
```
sms.id
sms.reference
```
and then to describe what each of these things are.